src/components/device/modals/DemoEndModalForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { List, Grid, Button, Divider, Image } from 'semantic-ui-react'
+import deviceimg from 'images/logo3.png'
+
+const features = [
+  {
+    header: 'Live statistics on processes data and results',
+    description: 'Multiple views and charts for a user to get a in depth understanding of an experiments in a aesthetic presentation.',
+    icon: 'chart line',
+  },
+  {
+    header: 'True control with phases',
+    description: 'Create a process from phases with individual sets of parameters and transitions to have optimal conditions for every single step of your bioprocess.',
+    icon: 'tasks',
+  },
+  {
+    header: 'Start a process from outside the lab',
+    description: 'The web applications allows you to run, monitor and edit processes from any device with an internet connection.',
+    icon: 'coffee',
+  },
+  {
+    header: 'User profiles',
+    description: 'Personalize your experience and save your favorite processes and benefit from a customizable user interface.',
```

Comment:
I may be commenting on something that Viktor already provided a suggestion for, but this might read more comfortably with commas. Oxford comma avoids the potential ambiguity.'
```suggestion
    description: 'Personalize your experience, save your favorite processes, and benefit from a customizable user interface.',
```

Reply:
maybe forget the oxford comma at the end because its not elsewhere